### DISCUSSION TITLE
Networking platform

### DISCUSSION THREAD

**Discussion by manuelalferez** [manuelalferez/chatcus#31] [Ideas]
I was thinking of a chat type channel where we can maintain our communications about Chatcus.

I have two option in mind: 
- [GitHub Discussions](https://github.com/manuelalferez/chatcus/discussions). Advantage: we keep everything in the same GitHub space
- [Gitter](https://gitter.im). 

What do you think?
@iamvishal345 @chalermporn @gitwikc @kritikash18

**Comment by iamvishal345** [comment]
Github Discussions is good for me

**Comment by gitwikc** [comment]
I mean, I've never really done any serious colab with a team, so Github Discussions might be a good starting point for me to learn some teamwork and coordination :wink:

**Comment by manuelalferez** [comment]
Great, GitHub Discussions then 😊
